### PR TITLE
[#9] Update SDK to fix panic on go1.19

### LINKED ISSUE #9: Update SDK
The same as https://github.com/nspcc-dev/neofs-http-gw/issues/188
Also consider rebasing

### PR DESCRIPTION
**Pull request by KirillovDenis**
close #9 
Signed-off-by: Denis Kirillov <denis@nspcc.ru>

### CODE REVIEW

**Comment by KirillovDenis** [comment]
The same problems with lint and vendor as https://github.com/nspcc-dev/distribution/pull/8#issue-1292859915

File: go.mod [hunk 1]
```diff
@@ -18,9 +18,9 @@ require (
 	github.com/gorilla/mux v1.8.0
 	github.com/mitchellh/mapstructure v1.1.2
 	github.com/ncw/swift v1.0.47
-	github.com/nspcc-dev/neo-go v0.98.2
-	github.com/nspcc-dev/neofs-sdk-go v1.0.0-rc.4.0.20220628114722-ab4d1e34a8ac
-	github.com/nspcc-dev/tzhash v1.5.2
+	github.com/nspcc-dev/neo-go v0.99.1
```
Comment (masterSplinter01) [inline go.mod:21]: Shouldn't we update to 99.2?

Comment (KirillovDenis) [reply go.mod:21]: SDK doesn't support neo-go v0.99.2 yet https://github.com/nspcc-dev/neofs-sdk-go/pull/328

Comment (KirillovDenis) [reply go.mod:21]: We don't use rpc client from `neo-go` which was changed in v0.99.2 so it doesn't matter.

**Review by alexvanin — APPROVED** [review]

**Pull request merged**